client/app/bundles/course/video/submission/containers/Statistics.jsx

```diff
@@ -24,21 +23,17 @@ const propTypes = {
   watchFrequency: PropTypes.arrayOf(PropTypes.number).isRequired,
 };
 
-class Statistics extends React.Component {
-  render() {
-    return (
-      <Tabs className={styles.statisticsGraphView}>
-        <Tab label="Frequency Graph">
-          <br />
-          <HeatMap watchFrequency={this.props.watchFrequency} />
-        </Tab>
-        <Tab label="Progress Graph">
-          <ProgressGraph sessions={this.props.sessions} />
-        </Tab>
-      </Tabs>
-    );
-  }
-}
+const Statistics = ({ watchFrequency, sessions }) => (
```

Comment:
Do you think we should just be consistent in just using 'props' here and deconstruct the props inside the function?

Reply:
Hmm also can, but this was done by the codemodding haha. It only deconstructed props that didn't result in an overly long first line.

I feel it's ok to have both, no need to particularly enforce either style.